source/_integrations/philips_js.markdown

```
@@ -24,7 +24,21 @@ ha_integration_type: integration
 
 The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.
 
-If your TV responds to `http://IP_ADDRESS_OF_TV:1925/system` then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
+
+## Differences between TVs 
+
+1. Some TVs use `https://`, other TVs use `http://`.
+2. Some TVs primarily use port 1926, other TVs use port 1925 only.
+
+## Check if this intergration can be used   
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)